### DISCUSSION TITLE
选择数学公式的样式为Tex，似乎和ISO没啥区别？是这样吗？

### DISCUSSION THREAD

**Discussion by WEIYujie1996** [sjtug/SJTUThesis#1057] [提问]
选择数学公式的样式为Tex，似乎和ISO没啥区别？是这样吗？

**Comment by LogCreative** [comment]
部分数学符号上会有区别，比如 `\Delta`, `\int` 等，还可以通过子选项 `uppercase-greek`、 `integral`、 `integral-limits` 分别设置，参见 [文档](https://mirrors.sjtug.sjtu.edu.cn/CTAN/macros/latex/contrib/sjtutex/sjtutex.pdf)。
